.github/workflows/docker.yml

```diff
@@ -70,7 +74,7 @@ jobs:
       - name: Build and push Docker image
         run: |
           echo $TAG
-          docker build . -f docker/Dockerfile -t ${TAG} --no-cache
+          docker build . -f docker/Dockerfile -t ${TAG}
```

Comment:
If we remove `--no-cache`, we cannot have the latest dependencies

Reply:
because we use `COPY` command, if something changes in your lmdeploy directory, it will always rebuild the layers from that command. see https://docs.docker.com/build/cache/#how-the-build-cache-works .
```
COPY . /opt/lmdeploy
```